handout/directives/structural_directives.md

```diff
@@ -1,6 +1,6 @@
 # Structural Directives
 
-Structural Directives are a way of handling how a component or element renders through the use of the `template` tag. This allows us to run some code that decides what the final rendered output will be. Angular 2 has a few built-in structural directives such as `ngIf`, `ngFor`, and `ngSwitch`.
+Structural Directives control what is added or removed from the DOM. It allows us to run some code that decides what the final rendered output will be. Angular 2 has a few built-in structural directives such as `ngIf`, `ngFor`, and `ngSwitch`.
```

Comment:
Take from Angular doc, under "Template Syntax" section, it say this, "When NgIf is false, Angular physically removes the element subtree from the DOM".

The changes follow the ng2 doc.